stepactions/eaas-get-ephemeral-cluster-credentials/0.1/eaas-get-ephemeral-cluster-credentials.yaml

```diff
@@ -76,3 +78,6 @@ spec:
     API_SERVER_URL=$("${OC[@]}" get cti "$CLUSTER_NAME" -o=jsonpath='{.status.apiServerURL}')
     echo "API Server URL: $API_SERVER_URL"
     echo -n "$API_SERVER_URL" > "$(step.results.apiServerURL.path)"
+    CONSOLE_URL=https://$("${OC[@]}" get route console -n openshift-console -o go-template --template="{{.spec.host}}")
```

Comment:
I tested this from a sample pipeline and ran into the following error which is to be expected:

```
step-get-kubeconfig
Found kubeconfig secret: cluster-ghqk7-admin-kubeconfig
Wrote kubeconfig to /credentials/cluster-ghqk7-kubeconfig
Found admin password secret: cluster-ghqk7-admin-password
Retrieved username
Wrote password to /credentials/cluster-ghqk7-password
API Server URL: https://ae2301db3888646799a6da0a74f33c56-7a5e7a3f524339df.elb.us-east-1.amazonaws.com:6443
Error from server (Forbidden): routes.route.openshift.io "console" is forbidden: User "system:serviceaccount:amisstea-25da3-eaas:namespace-manager" cannot get resource "routes" in API group "route.openshift.io" in the namespace "openshift-console"
```

This is trying to gather the console URL of the management cluster where the control plane is running. The desired route must instead be retrieved from the the ephemeral cluster using the credentials gathered above.

Reply:
oh. Could this be avoided by simply exporting KUBECONFIG as the previously exported kubeconfig?
This should set the oc call to the ephemeral cluster right?

Reply:
Yes, that's generally what needs to happen.

I also think we shouldn't reuse `${OC[@]}` for this call since that may include flags only intended for communicating to the management cluster (e.g. `--insecure-skip-tls-verify`).

Reply:
Will need to drop now and will come back tomorrow. If it doesn't work, could you please explain how I can test it so that I can solve it tomorrow first thing? Thanks

Reply:
It worked :smile: 

To test this I forked the minimal sample pipeline from the [docs](https://konflux.pages.redhat.com/docs/users/testing/cluster-provisioning.html#_example_pipelines) and modified this StepAction's git reference to point to your changes. Then I configured an IntegrationTestScenario for a sample Konflux component and ran a build to indirectly trigger it.